Unable to build docker image

When trying to build Docker image with `docker build .` I'm getting an error:
```
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.460 s
[INFO] Finished at: 2020-03-23T08:03:39Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.8:go-offline (default-cli): Goal requires a project to execute but there is no POM in this directory (/). Please verify you invoked Maven from the correct directory. -> [Help 1]
```

--- a/.github/workflows/docker-build.yml
+++ b/.github/workflows/docker-build.yml
@@ -1,0 +1,10 @@
+name: Docker build
+"on": [push, pull_request]
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
+      - name: Build
+        run: docker build .

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,12 +2,12 @@ FROM maven:3.6.3-jdk-13 as build_jar
 RUN yum install git -y
 RUN mkdir /jar
 COPY pom.xml /jar/pom.xml
-RUN mvn dependency:go-offline
+RUN cd /jar && mvn dependency:go-offline
 COPY src/ /jar/src/
 # @todo #40:30min Propagate git tag
 #  I.e. if docker image is based on version 1.5.0, the version should be propagated to the app jar
 #  file.
-RUN mvn package -B --quiet
+RUN cd /jar && mvn package -B --quiet
 
 FROM adoptopenjdk/openjdk13:alpine-jre
 LABEL description="Artipie is a binary repository managment tool."

